### PR TITLE
BXMSPROD-1468: switch all jobs to JDK 11

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

**JIRA**: [BXMSPROD-1468](https://issues.redhat.com/browse/BXMSPROD-1468) 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins retest this

**Review by almope — APPROVED** [review]

**Comment by mbiarnes** [comment]
Jenkins retest this

**Comment by almope** [comment]
Jenkins retest this

**Comment by mbiarnes** [comment]
Jenkins retest this

**Comment by mbiarnes** [comment]
Jenkins retest this

**Comment by mbiarnes** [comment]
Jenkins retest this

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
-def artifactsPath="/home/docker/kie-artifacts/$kieVersion"
 
 String EAP7_DOWNLOAD_URL = "http://download.devel.redhat.com/released/JBoss-middleware/eap7/7.3.0/jboss-eap-7.3.0.zip"
 
 // creation of folder
 folder("KIE")
 folder ("KIE/${baseBranch}")
-folder("KIE/${baseBranch}/daily-build")
-folder("KIE/${baseBranch}/docker")
+folder("KIE/${baseBranch}/daily-build-jdk1.8")
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:17]: we can shorten the name to just 8
```suggestion
folder("KIE/${baseBranch}/daily-build-jdk8")
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 2]
```diff
@@ -1,27 +1,22 @@
...
 
-def folderPath="KIE/${baseBranch}/daily-build"
-def dockerPath="KIE/${baseBranch}/docker"
+def folderPath = "KIE/${baseBranch}/daily-build-jdk1.8"
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:19]:
```suggestion
def folderPath = "KIE/${baseBranch}/daily-build-jdk8"
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 3]
```diff
@@ -140,19 +123,16 @@ pipeline {
...
                     "jbpmTestContainerMatrix" : {
-                        build job: "daily-build-${baseBranch}-jbpmTestContainerMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
+                        build job: "daily-build-jdk1.8-${baseBranch}-jbpmTestContainerMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
                     },
                     "kieWbTestsMatrix" : {
-                            build job: "daily-build-${baseBranch}-kieWbTestsMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
+                            build job: "daily-build-jdk1.8-${baseBranch}-kieWbTestsMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:132]:
```suggestion
                            build job: "daily-build-jdk8-${baseBranch}-kieWbTestsMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 4]
```diff
@@ -140,19 +123,16 @@ pipeline {
...
-                        build job: "daily-build-${baseBranch}-jbpmTestCoverageMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
+                        build job: "daily-build-jdk1.8-${baseBranch}-jbpmTestCoverageMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
                     },
                     "jbpmTestContainerMatrix" : {
-                        build job: "daily-build-${baseBranch}-jbpmTestContainerMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
+                        build job: "daily-build-jdk1.8-${baseBranch}-jbpmTestContainerMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:129]:
```suggestion
                        build job: "daily-build-jdk8-${baseBranch}-jbpmTestContainerMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 5]
```diff
@@ -140,19 +123,16 @@ pipeline {
             steps {
                 parallel (
                     "jbpmTestCoverageMatrix" : {
-                        build job: "daily-build-${baseBranch}-jbpmTestCoverageMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
+                        build job: "daily-build-jdk1.8-${baseBranch}-jbpmTestCoverageMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:126]:
```suggestion
                        build job: "daily-build-jdk8-${baseBranch}-jbpmTestCoverageMatrix", propagate: false, parameters: [[$class: 'StringParameterValue', name: 'kieVersion', value: kieVersion], [$class: 'StringParameterValue', name: 'baseBranch', value: baseBranch]]
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 6]
```diff
@@ -169,7 +149,7 @@ pipeline {
             emailext body: 'Build log: ${BUILD_URL}consoleText\\n' +
                            'Failed tests (${TEST_COUNTS,var="fail"}): ${BUILD_URL}testReport\\n' +
                            '(IMPORTANT: For visiting the links you need to have access to Red Hat VPN. In case you do not have access to RedHat VPN please download and decompress attached file.)',
-                     subject: 'Build #${BUILD_NUMBER} of daily builds ${baseBranch} branch FAILED',
+                     subject: 'daily-build-jdk1.8-${baseBranch} #${BUILD_NUMBER} FAILED',
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:152]:
```suggestion
                     subject: 'daily-build-jdk8-${baseBranch} #${BUILD_NUMBER} FAILED',
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 7]
```diff
@@ -179,13 +159,13 @@ pipeline {
                            'Failed tests (${TEST_COUNTS,var="fail"}): ${BUILD_URL}testReport\\n' +
                            '***********************************************************************************************************************************************************\\n' +
                            '${FAILED_TESTS}',
-                     subject: 'Build #${BUILD_NUMBER} of daily builds ${baseBranch} branch was UNSTABLE',
+                     subject: 'daily-build-jdk1.8-${baseBranch} #${BUILD_NUMBER} was UNSTABLE',
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:162]:
```suggestion
                     subject: 'daily-build-jdk8-${baseBranch} #${BUILD_NUMBER} was UNSTABLE',
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 8]
```diff
@@ -179,13 +159,13 @@ pipeline {
...
                      to: 'kie-jenkins-builds@redhat.com'
             cleanWs()         
         }
         fixed {
             emailext body: '',
-                 subject: 'Build #${BUILD_NUMBER} of daily builds ${baseBranch} branch is fixed and was SUCCESSFUL',
+                 subject: 'daily-build-jdk1.8-${baseBranch} #${BUILD_NUMBER} was SUCCESSFUL',
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:168]:
```suggestion
                 subject: 'daily-build-jdk8-${baseBranch} #${BUILD_NUMBER} was SUCCESSFUL',
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 9]
```diff
@@ -195,7 +175,7 @@ pipeline {
 }
 '''
 
-pipelineJob("${folderPath}/daily-build-pipeline-${baseBranch}") {
+pipelineJob("${folderPath}/daily-build-jdk1.8-pipeline-${baseBranch}") {
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:178]:
```suggestion
pipelineJob("${folderPath}/daily-build-jdk8-pipeline-${baseBranch}") {
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 10]
```diff
@@ -258,15 +238,13 @@ mv jbpm-$kieVersion/* .
 rm -rf jbpm-$kieVersion
 '''
 
-matrixJob("${folderPath}/daily-build-${baseBranch}-jbpmTestCoverageMatrix") {
+matrixJob("${folderPath}/daily-build-jdk1.8-${baseBranch}-jbpmTestCoverageMatrix") {
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:241]:
```suggestion
matrixJob("${folderPath}/daily-build-jdk8-${baseBranch}-jbpmTestCoverageMatrix") {
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 11]
```diff
@@ -331,16 +309,14 @@ mv jbpm-$kieVersion/* .
 rm -rf jbpm-$kieVersion
 '''
 
-matrixJob("${folderPath}/daily-build-${baseBranch}-jbpmTestContainerMatrix") {
+matrixJob("${folderPath}/daily-build-jdk1.8-${baseBranch}-jbpmTestContainerMatrix") {
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:312]:
```suggestion
matrixJob("${folderPath}/daily-build-jdk8-${baseBranch}-jbpmTestContainerMatrix") {
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 12]
```diff
@@ -416,7 +392,7 @@ rm sources.tar.gz
 mv kie-wb-common-$kieVersion/* .
 rm -rf kie-wb-common-$kieVersion'''
 
-matrixJob("${folderPath}/daily-build-${baseBranch}-kieWbTestsMatrix") {
+matrixJob("${folderPath}/daily-build-jdk1.8-${baseBranch}-kieWbTestsMatrix") {
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:395]:
```suggestion
matrixJob("${folderPath}/daily-build-jdk8-${baseBranch}-kieWbTestsMatrix") {
```

File: job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy [hunk 13]
```diff
@@ -515,12 +489,11 @@ rm sources.tar.gz
 mv droolsjbpm-integration-$kieVersion/* .
 rm -rf droolsjbpm-integration-$kieVersion'''
 
-matrixJob("${folderPath}/daily-build-${baseBranch}-kieServerMatrix") {
+matrixJob("${folderPath}/daily-build-jdk1.8-${baseBranch}-kieServerMatrix") {
```
Comment (mareknovotny) [inline job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy:492]:
```suggestion
matrixJob("${folderPath}/daily-build-jdk8-${baseBranch}-kieServerMatrix") {
```

File: job-dsls/jobs/seed_jobs/kie_new_branches_seed_job.groovy [hunk 14]
```diff
@@ -53,8 +53,9 @@ job("${folderPath}/a-seed-job-${baseBranch}") {
                     "job-dsls/jobs/kie/main/compile_downstream_build.groovy\n" +
                     "job-dsls/jobs/kie/main/prodProjects_downstream_production.groovy \n" +
                     "job-dsls/jobs/kie/main/upstream.groovy\n" +
-                    "job-dsls/jobs/kie/main/dailyBuild_pipeline.groovy\n" +
-                    "job-dsls/jobs/kie/main/dailyBuild_prod_pipeline.groovy\n" +
+                    "job-dsls/jobs/kie/main/dailyBuild_jdk1_8_pipeline.groovy\n" +
```
Comment (mareknovotny) [inline job-dsls/jobs/seed_jobs/kie_new_branches_seed_job.groovy:56]:
```suggestion
                    "job-dsls/jobs/kie/main/dailyBuild_jdk8_pipeline.groovy\n" +
```

**Pull request merged**